prime-router/docs/proposals/0017-metadata-tables-cleanup.md

```diff
@@ -0,0 +1,441 @@
+comment on column facility.facility_county_id is 'The ID of the county where the facility is located';
+comment on column facility.facility_state_id is 'The ID of the state where the facility is located';
+comment on column facility.facility_country_id is 'The ID of the country where the facility is located';
+comment on column facility.has_valid_clia is
+    'Indicates if the facility has a valid value for a CLIA. Some facilities will have a CLIA assigned by a PHA but is not actually valid';
+comment on column facility.facility_oid is
+    'The OID of the facility, if it exists. Typically an ISO identifier.';
+comment on column facility.facility_namespace_id is
+    'The namespace for the OID of the facility. Used in HD designators';
+comment on column facility.created_at is 'The timestamp for the creation of the record';
+comment on column facility.last_modified_at is 'The timestamp for when the record was last modified';
+
+
+-- our providers
+create table provider
+(
+    provider_id                         bigserial not null primary key
+    , provider_first_name               varchar(512) not null
+    , provider_last_name                varchar(512) not null
+    , provider_npi                      varchar(512) not null unique
+    , provider_callback_number          varchar(512) null
+    , provider_city                     varchar(512) null
+    , provider_postal_code              varchar(50) null
+    , provider_county_id                bigint null references county(county_id)
+    , provider_state_id                 bigint null references state(state_id)
+    , provider_country_id               bigint null references country(country_id)
+    , created_at                        timestamp default current_timestamp
+    , last_modified_at                   timestamp default current_timestamp
+);
+
+comment on table provider is
+    'The provider related to the testing process. This could be an actual medical provider, but could also be a business or a non-traditional testing site.';
+comment on column provider.provider_id is
+    'The primary key for the provider table';
+comment on column provider.provider_first_name is
+    'The provider first name';
+comment on column provider.provider_last_name is
+    'The provider last name';
+comment on column provider.provider_npi is
+    'The NPI for the provider';
+comment on column provider.provider_callback_number is
+    'The callback number for the provider';
+comment on column provider.provider_city is
+    'The city for the provider';
+comment on column provider.provider_postal_code is
+    'The postal code of the provider';
+comment on column provider.provider_county_id is
+    'The county ID for the provider';
+comment on column provider.provider_state_id is
+    'The state ID for the provider';
+comment on column provider.provider_country_id is
+    'The country ID for the provider';
+comment on column provider.created_at is 'The timestamp for the record creation';
```

Comment:
what would modify a provider record in this table? Would last_modified ever be something other than created_at?